book/website/reproducible-research/open/open-source.md

```diff
@@ -1,237 +1,21 @@
 (rr-open-source)=
-# Open Source Software
+# Open Source
 
-(rr-open-source-whatis)=
-## What Is Open Source Software?
+## Definition
 
-When a project is open-source [{term}`def<Open Source Software>`], anybody can view, use, modify, and distribute the project for any purpose.
-These permissions are enforced through an open-source licence.
-Open source is powerful because it lowers the barriers to adoption, allowing ideas to spread quickly.
-In its most basic form, open-sourcing your software means putting your code online where it can be viewed and reused by others.
+### Noun
 
-Many of the most widely used research software is open source.
-Perhaps the paradigmatic example is the scikit-learn Python package for machine learning (Pedregosa et al., 2011), which, in the space of just over five years, has attracted over 500 unique contributors, 20,000 individual code contributions, and 2,500 article citations.
-Producing a comparable package using a traditional closed-source approach would likely not be feasible.
-It would, at the very least, require a budget of tens of millions of dollars.
-While scikit-learn is an outlier, hundreds of other open-source packages that support much more domain-specific needs depend similarly on unsolicited community contributions; for example, the NIPY (neuroimaging in Python) group of projects in neuroimaging (Gorgolewski et al., 2016).
-Notably, such contributions not only result in new functionality from which the broader community can benefit, but also regularly provide their respective authors with greater community recognition, and lead to new project and employment opportunities.
+Practice of providing digital resources with their sources.
 
-Researchers that make use of open-source software often make changes to them, such as adding features they need for their research, or fixing bugs.
-They can then contribute these improvements back to the main project so the wider community can take advantage of them.
+### Adjective
 
-(rr-open-source-benefitsyou)=
-## How Running and Contributing to Open-Source Software Projects Benefits You
+Relating to a digital resource whose source files are available.
```

Comment:
I think even this might be too specific. Does something have to be digital?

Reply:
I would think of a blueprint/object design printed on paper, the idea of sources may be used in other context to say where something come from (even journalism).

Maybe it depends if it's in a library with public access ! I usually prefer to remain in digital artefacts, but I don't know what may be the answer to this question. Is openness a digital thing ?

Reply:
> (Yes, we're aware of the specialized meaning "open source" has in the intelligence community. This is a feature, not a bug.)  
> **Eric Raymond** in the article « [Goodbye, "free software"; hello, "open source"](http://www.catb.org/~esr/open-source.html) », near(/in) 1998.

OSINT was around since the formalization of the concept, one more aspect to the meaning (which lack to the traditional definition(s)/explanations). A little more exotic, I've seen used the idea of open source education where you can have both an understanding of an education with open source resources, but which could also be more about an education from freely available information for some people (not necessarily open licensed, more like the open source of OSINT).

It's linked again in some extent to the boundaries of openness, to whether open (source) starts as soon as the components are visible or when you have more or less rights/technical abilities over them.

The term "open" is polysemous and ambiguous, the term "source" is polysemous and ambiguous, the term "open source" is even more polysemous and ambiguous. I think it's normal that not everything is crystal clear !

Reply:
Hey both – just flagging here that the methods preceding/that developed into OSINT - as in using information in the public domain for investigative and/or intelligence purposes – has much longer roots than 'open source / public access information'... and does not necessarily overlap with 'open (source) education' for a variety of reasons. It's definitely a 'dark knight' of the broader landscape!

Reply:
You think anterior to computers and the end of WW2, like practices done by research agencies or the Sicherheitsdienst ? 50's being a more appropriate date than 90's for the origin of what we call nowadays "open source". OSINT would be clearly an important additional dimension to consider capturing the full meaning of open source !

I have to admit that it was uncertain starting from the definition of open source, but I guess this is a well deserved

![godwin-s-law-adolf-hitler-internet-forum-blog-islamic-chef-a096b2423d5e84a76c103ce5e1dec6e4](https://github.com/the-turing-way/the-turing-way/assets/34010605/76d9dfbe-b47e-44f1-9def-4600a80b763c)

Reply:
Hi @AbcSxyZ , the graphic and comment including inappropriate image used to make your point is in direct breach of Code of Conduct. I have deleted your last post. I remind you to please follow and uphold our Code of Conduct in all your interactions and communications within The Turing Way: https://github.com/the-turing-way/the-turing-way/blob/main/CODE_OF_CONDUCT.md